docs/sources/configuration/_index.md

```diff
@@ -1704,11 +1722,11 @@ background:
 memcached:
   # Configures how long keys stay in memcached.
   # CLI flag: -<prefix>.memcached.expiration
-  expiration: <duration>
+  expiration: <duration> | default = 0
```

Comment:
I don't think this is true. I believe it defaults to `default_validity`, which by default is 1 hour. Only when `default_validity` is set to 0 will this default to 0.

Reply:
you are right, I got this config from [here](https://github.com/grafana/loki/blob/cd9d9c9b425e0ad11d5510deeb8fc2ff76c7a4aa/pkg/storage/chunk/cache/memcached.go#L32)
but looks like its updated [here](https://github.com/grafana/loki/blob/cd9d9c9b425e0ad11d5510deeb8fc2ff76c7a4aa/pkg/storage/chunk/cache/cache.go#L101)